cni/pkg/repair/options.go

```diff
@@ -73,6 +58,7 @@ func parseFlags() (filters *repair.Filters, options *ControllerOptions) {
 	pflag.String("field-selectors", "", "A set of field selectors in label=value format that will be added to the pod list filters")
 
 	// Repair Options
+	pflag.Bool("enabled", true, "Whether enable race condition repair or not.")
```

Comment:
is `enabled` scoped in some way to indicate its for repair?

Reply:
Yeah it has `viper.SetEnvPrefix("REPAIR")` call underneath, but actually I am not sure the implication of setting that since install is also using viper. I updated to remove that and attach `repair` prefix to all flags directly.